### PR TITLE
Add db connection when deleting old records

### PR DESCRIPTION
**Pull request by MaksTkachuk**

### CODE REVIEW

File: api_tests/db/delete_old_records.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from ..models.queries import delete_records_older_than
```
Comment (kalkov) [inline api_tests/db/delete_old_records.py:1]: `db` is used for files dealing with connection and migration. 
We don't have similar file in the code base, so I would suggest to put it into [api_test folder](https://github.com/santiment/api-tests/tree/master/api_tests)

File: cli.py [hunk 2]
```diff
@@ -14,6 +14,7 @@
 from api_tests.statuspage.metrics import run as run_metrics
 from api_tests.statuspage.sanbase_webapp import run as run_sanbase_webapp
 from api_tests.db.migrations import run_migrations
+from api_tests.db.delete_old_records import delete_old_records as delete_records
 from api_tests.models.queries import delete_records_older_than
```
Comment (kalkov) [inline cli.py:18]: leftover

File: cli.py [hunk 3]
```diff
@@ -99,12 +98,8 @@ def migrate_db():
 
 def delete_old_records():
     """Delete records from the DB that are older than DATABASE_RETENTION_PERIOD"""
-    cutoff_date = dt.utcnow() - td(days=DATABASE_RETENTION_PERIOD_DAYS)
-    cutoff_date_str = dt.strftime(cutoff_date, DT_FORMAT)
-    logging.info('Connecting to database...')
-    db.connect(reuse_if_open=True)
-    logging.info(f"Deleting records from the DB older than {cutoff_date_str}")
-    delete_records_older_than(cutoff_date)
+    logging.info("Deleting old records from the DB")
```
Comment (kalkov) [inline cli.py:101]: Would be handy to print "Deleting records older than {DATABASE_RETENTION_PERIOD_DAYS} days"

Comment (MaksTkachuk) [reply cli.py:101]: done

File: api_tests/db/delete_old_records.py (hunk 1 above)
Comment (MaksTkachuk) [reply api_tests/db/delete_old_records.py]: done

File: cli.py (hunk 3 above)
Comment (kalkov) [reply cli.py:99]: Add the logging here, not in delete_records file

**Review by kalkov — APPROVED** [review]

**Pull request merged**